cli/src/commands/download.ts

```diff
@@ -0,0 +1,91 @@
+import { CLIError } from '@oclif/errors';
+import * as path from 'path';
+import * as tar from 'tar';
+import * as request from 'request';
+import { spawn } from '@binaris/utils-subprocess';
+import Command from '../utils/command';
+import { Project } from '../utils/helpers';
+
+export default class Download extends Command {
+  public static description = 'download application';
+  public static examples = [
+    `$ ${Command.cliBinName} download 123456`,
+  ];
+
+  public static args = [
+    {
+      name: 'ID',
+      required: true,
+      description: 'application id',
+    },
+  ];
+
+  public static strict = true;
+
+  public async run() {
+    const { args } = this.parse(Download);
+    const { ID: applicationId } = args;
+    await this.authenticate();
+    const applications = await this.lycanClient.listApps();
```

Comment:
Nit: This should use some new API, `describeApp`.

Reply:
indeed. this is being worked on.